superset-frontend/plugins/plugin-chart-echarts/src/controls.tsx

```diff
@@ -87,7 +87,7 @@ const legendOrientationControl: ControlSetItem = {
     ],
     default: legendOrientation,
     renderTrigger: true,
-    description: t('Legend type'),
+    description: t('Orientation'),
```

Comment:
Thanks for the fix! Just one nit:
```suggestion
    description: t('Legend Orientation'),
```